.gitattributes

```diff
@@ -0,0 +1,4 @@
+.travis/ export-ignore
+export/ export-ignore
+test/ export-ignore
```

Comment:
`.gitattributes` are useful for libraries, but not for applications?

Plus I think not having the `test` folder means you couldn't actually build it (since the build process runs the tests).

Reply:
Would exclude `test/` then; `export/` becomes an empty folder so it doesn't matter very much; `.travis` doesn't harm anyone. So I'll drop this.